wpiutil/.styleguide

```diff
@@ -11,87 +11,75 @@ cppSrcFileInclude {
 }
 
 generatedFileExclude {
+  src/main/native/third_party/llvm/cpp/llvm/
+  src/main/native/third_party/llvm/include/wpi/AlignOf\.h$
```

Comment:
Is there a reason this can't just be everything in `src/main/native/third_party/llvm/include/wpi/`?

Reply:
I left it the way that it was before. Just changed the order to group all the third party together.